src/environment.js

```diff
@@ -122,9 +122,10 @@ if (process.env.NODE_ENV !== 'production') {
 
 export const defaultEthNode =
   getDefaultEthNode() || networkConfig.nodes.defaultEth
+
 export const web3Providers = {
-  default: new Web3.providers.WebsocketProvider(defaultEthNode),
-  wallet: window.web3 && window.web3.currentProvider,
+  default: provider(defaultEthNode),
```

Comment:
Not sure we should use `eth-provider` for the default provider yet; I've seen some odd reconnects happening that cause a transaction intent to not show up (would be really, really confusing for the user).